src/styles/shared/_breakpoints.scss

```diff
@@ -18,7 +18,7 @@ $not-condensed-min-page: $not-condensed-content + $not-condensed-outer-spacing;
 $partially-condensed-min-page: $partially-condensed-content +
   $partially-condensed-outer-spacing;
 
-$nav-size: em(layout-width(nav));
+$nav-size: calc(var(--p-nav-width) / 16 * 1em);
```

Comment:
For the [same reason](https://github.com/Shopify/polaris-react/pull/3273/files#r491062443), I'm doing it here.